torch/ao/quantization/fx/README.md

```diff
@@ -0,0 +1,407 @@
+# FX Graph Mode Quantization Design Doc
+High Level FX Graph Mode Quantization Flow
+
+float_model            QConfigMapping           BackendConfig
+    \                          |                        /
+     \                         |                      /
+      \                        |                    /
+(prepare_fx/prepare_qat_fx)                        /
+—-------------------------------------------------------
+|                         Fuse                         |
+|                  QAT Module Swap                     |
+|                 Insert Observers                     |
+—-------------------------------------------------------
+                              |
+                      Calibrate/Train
+                              |
+(convert_fx)                  |
+—--------------------------------------------------------
+|                         Convert                       |
+|                        Lowering                       |
+—--------------------------------------------------------
+                              |
+                       Quantized Model
+
+Please refer to TODO: link [(draft) README.md of quantization syntax transforms] for definitions of terminologies.
+
+## Overview
+The FX graph representation is pretty close to python/eager mode, it preserves many python/eager mode constructs like modules, functionals, torch ops, so overall the implementation reuses some of building blocks and utilities from eager mode quantization, this includes the QConfig, QConfig propagation (might be removed), fused modules, QAT module, quantized modules, QAT module swapping utility. Also the overall flow exactly matches eager mode quantization, the only difference is that the transformations like fusion, inserting stubs are fully automated and controlled by QConfigMapping and BackendConfig.
+
+### Quantization Flow Code Example
```

Comment:
should we delete this, since it's covered by user docs?

Reply:
sure, sounds good